src/client/index.js

```diff
@@ -70,6 +70,24 @@ module.exports = class Client {
     });
   }
 
+  findWithPath(path, filters, params) {
```

Comment:
Should't this be  a bit more generic method like `query(method = 'GET', path, filters, params)` ?